russh/src/client/session.rs

```diff
@@ -319,6 +319,56 @@ impl Session {
         }
     }
 
+    /// Requests a UDS forwarding from the server, `socket path` being the server side socket path.
+    ///
+    /// If `reply_channel` is not None, sets want_reply and returns the server's response via the channel,
+    /// [`Some<String>`] for a success message with the client side socket path, [`None`] for failure.
+    pub fn streamlocal_forward(
+        &mut self,
+        reply_channel: Option<oneshot::Sender<Option<String>>>,
+        socket_path: &str,
+    ) {
+        if let Some(ref mut enc) = self.common.encrypted {
+            let want_reply = reply_channel.is_some();
+            if let Some(reply_channel) = reply_channel {
+                self.open_global_requests.push_back(
+                    crate::session::GlobalRequestResponse::StreamLocalForward(reply_channel),
+                );
+            }
+            push_packet!(enc.write, {
+                enc.write.push(msg::GLOBAL_REQUEST);
+                enc.write.extend_ssh_string(b"streamlocal-forward");
```

Comment:
Should be namespaced to @openssh.com: https://github.com/openssh/openssh-portable/blob/master/PROTOCOL#L355

Reply:
Done, streamlocal-forward => streamlocal-forward@openssh.com and the same for cancel-streamlocal-forward